### ISSUE TITLE
No tool call and llm call recorded with trace_agent.

### ISSUE THREAD

**Issue by poojatambe** [raga-ai-hub/ragaai-catalyst#26]
Hi,

Thanks for your work.
I have used trace agent decorator, but getting empty tool call and llm call tracing.
![image](https://github.com/user-attachments/assets/9665139e-a604-4e52-8f5d-6f6f0d1c3450)

How to define tool_call and llm_call for getting logs in agent tracing?

Code:
```
@tracer.trace_agent('word_length_agent')
def word_length(message, tool_call=None, llm_call=None):
    tool = {
        "type": "function",
        "function": {
            "name": "get_word_length",
            "description": "Get the word length",
            "parameters": {
                "type": "object",
                "properties": {
                    "word": {
                        "type": "string",
                        "description": "Any word",
                    }
                },
                "required": ["word"],
            },
        },
    }   
    output = word_extraction_llm(message, tool)
    arguments = json.loads(
        output.choices[0].message.tool_calls[0].function.arguments
        )
    length = get_word_length(arguments.get('word'))
    print("Length of the word {}: ".format(arguments.get('word')), length)
    return length
```

**Comment by kiranscaria** [comment]
@poojatambe Thanks for reporting the issue. 

We have added the fix to it in Release [v1.1.beta.0 ](https://github.com/raga-ai-hub/AgentNeo/releases/tag/1.1).

PR: #33

**Comment by kiranscaria** [comment]
Closing the issue for now.Please reopen in case the issue persists.
